tests/utils/files/report_columns_select_column.yaml

```diff
@@ -0,0 +1,67 @@
+columns:
+    Analyzable:
+        from: 'function'
+        name: "get_read_level"
+        variables:
+         - levels
+         - depth
+        column: 1
+    Min_read_depth300:
```

Comment:
200

Reply:
let's add it to our actual pipeline instead of changing the tests